src/main/features/response/form/models/statement-of-means/residence.ts

```diff
@@ -0,0 +1,6 @@
+export class Residence {
```

Comment:
what is this class for?

Reply:
This is class that @rlewan is implementing, but as we were developing it simultaneously I added it. This will be replaced by his code.